package.json

```diff
@@ -19,9 +19,13 @@
     "compression": "^1.5.2",
     "errorhandler": "^1.4.3",
     "express": "^4.13.3",
+    "express-minify": "^0.2.0",
     "express-sslify": "^1.0.1",
-    "jstransformer-uglify-js": "^1.1.1",
+    "fs-walk": "0.0.1",
+    "html-validator": "^1.0.0",
+    "htmlencode": "0.0.4",
     "js-yaml": "^3.4.2",
+    "jstransformer-uglify-js": "^1.1.1",
```

Comment:
It was there sure because we used it before these changes :P

On Sat, Aug 6, 2016 at 10:36 AM, Joshua Mervine <notifications@github.com>
wrote:

> In package.json
> <https://github.com/MaxCDN/bootstrap-cdn/pull/733#discussion_r73782470>:
>
> >      "js-yaml": "^3.4.2",
> > +    "jstransformer-uglify-js": "^1.1.1",
>
> Actually it was already there just higher up, but probably not. As I said,
> I'll verify.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MaxCDN/bootstrap-cdn/pull/733/files/ac05316354f9e2cb784b3878ccc539220d5d0d81#r73782470>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtQ7I-PjhISK0DgbPisKkoZcJAwheks5qdDmXgaJpZM4JeOU8>
> .
>